components/arm/arm.go

```diff
@@ -43,17 +40,6 @@ func init() {
 // SubtypeName is a constant that identifies the component resource API string "arm".
 const SubtypeName = "arm"
 
-// MTPoob is a string that all MoveToPosition errors should contain if the method is called
-// and there are joints which are out of bounds.
-const MTPoob = "cartesian movements are not allowed when arm joints are out of bounds"
-
-var (
-	defaultLinearConstraint  = motionplan.LinearConstraint{}
-	defaultArmPlannerOptions = &motionplan.Constraints{
-		LinearConstraint: []motionplan.LinearConstraint{defaultLinearConstraint},
```

Comment:
I need to add this back in before merging

Reply:
We want these to go to the motion service, right?

Reply:
Probably where they should live, since there's no way to configure them through the arm drivers.